index.html

```diff
@@ -30,16 +45,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div id="box1" class="box">Box 1</div>
+                <div id="box2" class="box">Box 2</div>
+                <div id="box3" class="box">Box 3</div>
+                <div id="box4" class="box">Box 4</div>
+                <div id="box5" class="box">Box 5</div>
+                <div id="box6" class="box">Box 6</div>
+                <div id="box7" class="box">Box 7</div>
+                <div id="box8" class="box">Box 8</div>
+                <div id="box9" class="box">Box 9</div>
+                <div id="box10" class="box">Box 10</div>
```

Comment:
Literally the ONLY think I have to nit pick about is this. Instead of making an id for each box, you can just make a class and add it to the box class.
Something like
```
<div class="box box1">Box 1</div>
<div class="box box2">Box 2</div>
<div class="box box3">Box 3</div>
```
And then you can just style it that way

Your way works too and it's not wrong. But I would only use an id sparingly. If you have a class and all you are doing is updating that class. Instead of making the changes using an id, update the class using another class.